### PR TITLE
Create a linter script and fix errors.

### PR DESCRIPTION
**Pull request by Kashomon**
This PR creates a golint script and fixes some errors.

Next steps: some slight modifications for prow, and then create a new presubmit script.

Based on https://github.com/kubernetes-sigs/kind/blob/master/hack/verify-golint.sh

### CODE REVIEW

File: pkg/converter/unstructured.go [hunk 1]
```diff
@@ -30,6 +30,7 @@ type UnstructuredConverter struct {
 	o *unstructured.Unstructured
 }
 
+// ExtractObjectMeta creates an ObjectMeta object from an Unstructured object.
```
Comment (ihmccreery) [inline pkg/converter/unstructured.go]: I'm surprised this doesn't exist somewhere in K8s libraries.  Have you checked?

File: pkg/inline/path_rewriter.go [hunk 2]
```diff
@@ -31,7 +31,12 @@ type PathRewriter interface {
 // RelativePathRewriter rewrites paths when the paths are relative paths.
 type RelativePathRewriter struct{}
 
+// RewriteObjectPath rewrites object paths if the path is relative.
```
Comment (ihmccreery) [inline pkg/inline/path_rewriter.go]: Rewrites from what to what?

File: hack/verify-golint.sh [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+REPO_ROOT=$(git rev-parse --show-toplevel)
+cd "${REPO_ROOT}"
+
+command -v golint >/dev/null 2>&1 || {
+  errmsg="Error: golint is required. run 'go install github.com/golang/golint'"
+  printf >&2 "${errmsg}"; exit 1;
+}
+
+golint -set_exit_status ./pkg/...
```
[the remark this answers is not in the document]
Comment (Kashomon) [reply hack/verify-golint.sh]: Done.

File: pkg/commands/cmdlib/bundleio.go [hunk 4]
```diff
@@ -91,7 +91,8 @@ func inlineData(ctx context.Context, data *bundle.Bundle, rw files.FileReaderWri
 	return data, nil
 }
 
-// WriteContentsStructured writes some structured contents. The contents must be serializable to both JSON and YAML.
+// WriteStructuredContents writes some structured contents. The contents must
```
[the remark this answers is not in the document]
Comment (Kashomon) [reply pkg/commands/cmdlib/bundleio.go]: Added a comment about obj. What would you like to know about rw and g? I feel like the types tell what they are

File: pkg/commands/cmdlib/bundleio.go [hunk 5]
```diff
@@ -100,7 +101,8 @@ func WriteStructuredContents(ctx context.Context, obj interface{}, rw files.File
 	return WriteContents(ctx, g.OutputFile, bytes, rw)
 }
 
-// WriteContents writes some bytes to disk or stdout. if outPath is empty, write to stdout instdea.
+// WriteContents writes some bytes to disk or stdout. if outPath is empty,
```
[the remark this answers is not in the document]
Comment (Kashomon) [reply pkg/commands/cmdlib/bundleio.go]: Done

File: pkg/commands/cmdlib/bundleio.go [hunk 6]
```diff
@@ -100,7 +101,8 @@ func WriteStructuredContents(ctx context.Context, obj interface{}, rw files.File
...
+// write to stdout instdea.
```
[the remark this answers is not in the document]
Comment (Kashomon) [reply pkg/commands/cmdlib/bundleio.go]: Done

File: pkg/converter/unstructured.go (hunk 1 above)
Comment (Kashomon) [reply pkg/converter/unstructured.go]: I did check. I spent a long time looking at https://github.com/kubernetes/apimachinery/blob/master/pkg/apis/meta/v1/unstructured/unstructured.go and neighboring directories, but couldn't find anything.

File: pkg/inline/path_rewriter.go (hunk 2 above)
Comment (Kashomon) [reply pkg/inline/path_rewriter.go]: Added an example.

**Comment by Kashomon** [comment]
Thanks!

**Pull request merged**